<?xml version="1.0"?>

<launch>

	<!-- Start ipa_graspdata_processor node -->
	<include file="$(find ipa_graspdata_processor)/ros/launch/graspdata_processor_offline.launch" />

	<!-- Start ipa_grasp_pose_manager node -->
	<node ns="ipa_graspdata_manager" name="ipa_graspdata_manager_offline" pkg="ipa_graspdata_manager" type="ipa_graspdata_manager" output="screen" >
		<rosparam command="load" file="$(find ipa_graspdata_manager)/ros/launch/graspdata_manager_settings.yaml" />
		<param name="process_raw_database" value="true" />
	</node>

</launch>
